index.d.ts

```diff
@@ -84,7 +110,7 @@ declare namespace pug {
   /** 
    * An object that can have multiple properties of any type.
    */
-  export interface AnyObject {
+  export interface LocalsObject {
```

Comment:
`AnyObject` isn't meaningful - you don't know what is the function of object you are passing to as a param of `render()` or other function. `LocalsObject` means it's the object holding locals variables to render the view from template.